wg-ml-experience/charter.md

```diff
@@ -0,0 +1,64 @@
+    - Pipelines;
+    - Kubeflow Trainer
+    - Kubeflow Katib
+    - Model Registry;
+    - Model Serving (KServe);
+    - Feast.
+
+2. Promote the reusability of UI components from other Kubeflow UIs into the IDE (e.g., rendering a pipeline graph inside the JupyterLab environment) by establishing a shared contract between the IDE WG and the wider Kubeflow community. 
+
+3. Develop a Python SDK to simplify operationalization across Kubeflow components and provide a “one-stop-shop” for practitioners who want easy access to Kubeflow services. The SDK also provides the groundwork for the IDE extension automation and workflows.
+    - Create a single installation and configuration layer for users interacting programmatically with the Kubeflow ecosystem via SDKs.
+    - The “common” SDK is not meant to replace individual components’ SDKs but rather to offer a unified access layer to simplify dependency management and shared configuration (like authorization).
+
+#### Guiding Principles
+
+- Synergy among Kubeflow Working Groups: Collaborate with other WG to promote reusability of UI components from other Kubeflow UIs to create a single UX between Jupyter IDE and Kubeflow Central Dashboard;
+- Collaboration with other open-source IDE projects (like Jupyter and VSCode) to promote the creation and reusability of open standards for AI/ML tools (protocols, communication exchange, file formats, etc.) and plugins. The aim of this group is to actively participate in the development of these standards to include Kubeflow in a broader ecosystem or interoperable tools. 
+
+#### Cross-cutting and Externally Facing Processes
+
+- Collaboration with other Kubeflow WGs, including WG Notebooks, WG Pipelines, WG Training, WG AutoML, WG Data, and WG Serving, ensures that IDE tools are interoperable across different stages of the ML lifecycle.
+- Coordination with the release teams to align updates in IDE tools with broader Kubeflow release schedules.
+
+
+### Out of scope
+
+- Building and maintaining Notebook/Workspaces images (this falls under the WG Notebooks).
+
+## Roles and Organization Management
+
+This WG adheres to the Roles and Organization Management outlined in [wg-governance] and opts-in to updates and modifications to [wg-governance].
+
+### Additional responsibilities of Chairs
+
+- Coordinating and facilitating discussions on ML experience topics in scope of the WG, within the WG itself and the Kubeflow community.
+- Ensuring alignment with overall Kubeflow goals and objectives in the context of user experience to data scientists and machine learning practitioners on Kubeflow.
+
+### Additional responsibilities of Tech Leads
```

Comment:
I think this whole chair vs tech lead is too much governance overhead.

Reply:
@juliusvonkohout I agree with you because I prefer to always push towards a 'flat' community structure. The reason why this is here is because I copied it from a template. Are the steering committee ok if I remove it? We can always introduce this concept in the future if necessary.

Reply:
Leads and Chairs have different responsibilities according to this: https://github.com/kubeflow/community/blob/master/wgs/wg-governance.md#chair 
Check this for the history around WG proposal: https://github.com/kubeflow/community/pull/348

Today, we face a challenge due to a lack of sufficient Kubeflow contributors, resulting in most working groups having the same leads and chairs.

Reply:
I agree @juliusvonkohout.

I'm happy to be tech lead for the SDK if needed, but, frankly, users don't care and we should just focus on what users want which is a good MLX.